.github/workflows/aws-experiment-pipeline.yml

```diff
@@ -122,19 +122,25 @@ jobs:
       - name: "[Debug]: check chaos resources"
         if: ${{ failure() }}
         continue-on-error: true
+        env:
```

Comment:
May be we can provide this env at job level, so that we don't to provde it in all steps?